vital/types/database_query.h

```diff
@@ -45,6 +45,7 @@
 #include <vital/vital_config.h>
 
 #include <memory>
+#include <stdexcept>
```

Comment:
Did this need to be added to the .h instead of the .cxx?

Reply:
I would recommend moving this include to the implementation since it is not needed in the interface definition.

Reply:
Yeah my mistake. I'll push this change right away